From 61260846deb7a5ad6e0d240fe4e2fb6e8936bb79 Mon Sep 17 00:00:00 2001
From: Marc Mutz <marc.mutz@qt.io>
Date: Wed, 24 Sep 2025 08:15:13 +0200
Subject: [PATCH 206/553] Mark qbytedata_p.h as security-critical

It's only used in QtNetwork and QtHttpServer, which already makes it
critical, and its implementation is far from trivial, so it cannot be
considered significant-only.

QUIP: 23
Task-number: QTBUG-135195
Task-number: QTBUG-135564
Pick-to: 6.8
Change-Id: I69f5788c8b03f706b6669079c8aa561084c826b6
Reviewed-by: Ivan Solovev <ivan.solovev@qt.io>
(cherry picked from commit e3e4ec558d83b4e958892aa57d042fbae6af8304)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/text/qbytedata_p.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/corelib/text/qbytedata_p.h b/src/corelib/text/qbytedata_p.h
index fcbf82d6ceb..fd791b04385 100644
--- a/src/corelib/text/qbytedata_p.h
+++ b/src/corelib/text/qbytedata_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:critical reason:data-parser
 
 #ifndef QBYTEDATA_P_H
 #define QBYTEDATA_P_H
-- 
2.51.2

